repondeur/zam_repondeur/templates/table_detail.html

```diff
@@ -9,7 +9,48 @@
     .box {
         margin: 0 3rem 2rem 3rem;
         padding: 1rem;
+        position: relative;
     }
+    .menu {
+        position: absolute;
+        top: 8px;
+        right: 8px;
+    }
+        .menu .options {
+            display: block;
+            width: 1.5rem;
+            height: 1.5rem;
+            background-image: url("{{ request.static_url('zam_repondeur:static/img/options.svg') }}");
+            background-position: center;
+            background-repeat: no-repeat;
+        }
+        .menu .options:hover,
+        .menu.open .options {
+            background-color: white;
+            border: 1px solid #CCC;
+            border-radius: 4px;
+        }
+        .menu .details {
+            display: none;
+            position: absolute;
+            top: 25px;
+            right: 0;
+            background-color: white;
+            padding: 0.5rem 1rem;
+            border: 1px solid #CCC;
+            border-radius: 4px;
+            box-shadow: 4px 4px 2px #EEE;
+        }
+        .menu.open .details {
+            display: block;
+        }
+        .menu .details nav {
+            display: flex;
+            flex-direction: column;
+        }
+        .menu .details nav input.text {
+            -webkit-appearance: ;
```

Comment:
Might require a comment to be more explicit?

Reply:
Oops, that should not be there!